pkg/apis/servicecatalog/types.go

```diff
@@ -445,6 +439,12 @@ type ServiceInstanceSpec struct {
 	// end-user. User-provided values for this field are not saved.
 	// +optional
 	UserInfo *UserInfo
+
+	// UpdateRequests is a strictly increasing, non-negative integer counter that
+	// can be manually incremented by a user to manually trigger an update. This
+	// allows for parameters to be updated with any out-of-band changes that have
+	// been made to the secrets from which the parameters are sourced.
+	UpdateRequests int64
```

Comment:
if it's non-negative as the comment says, shall we use the `uint64` type?

Reply:
From https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#primitive-types

> Do not use unsigned integers, due to inconsistent support across languages and libraries. Just validate that the integer is non-negative if that's the case.

Reply:
thanks